scripts/check-governance-evidence-index.py

```
@@ -29,36 +28,37 @@ class WorkflowReference:
     text = (
-        text.replace('**', '')
-        .replace('__', '')
-        .replace('`', '')
-        .replace('’', "'")
-        .replace('‘', "'")
-        .replace('“', '"')
-        .replace('”', '"')
+        text.replace("**", "")
+        .replace("__", "")
+        .replace("`", "")
+        .replace("’", "'")
```

Ruff (0.15.12)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)